### PR TITLE
Added contact form section

### PR DESCRIPTION
**Pull request by mansa-susa**
- Added the form to the contact section
- Implemented client-side verification
- Added style declarations for the contact section

### CODE REVIEW

**Comment by mansa-susa** [comment]
- Added background image to the contact section
- Coding partner styled span text in footer

File: index.html [hunk 1]
```diff
@@ -144,57 +128,84 @@ <h3>Multi-Post Stories Gain+Glory</h3>
...
 
+        <form action="https://formspree.io/f/xvolveaz" method="post">
+          
+          <ul>
+            <li><input type="text" name="first_name" placeholder="Full name" required maxlength="30"/></li>
+            <li><input type="email" name="email" placeholder="Email address" required/></li>
+            <li>
+            <textarea required maxlength="500">Enter text here</textarea>
+            </li>
+            <li><button type="submit">Get in touch</button></li>
```
Comment (awais-amjed) [inline index.html:192]: Kindly implement button interactions as shown in the Figma design 🤗

image

File: style.css [hunk 2]
```diff
@@ -221,6 +221,18 @@ i.hamburger {
...
 }
 
+footer span {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #4252;
```
Comment (awais-amjed) [inline style.css]: Kindly change this color according to Figma design to make the email in your footer visible 🤗

Your design:

image

Figma design:

image

File: index.html [hunk 3]
```diff
@@ -144,57 +128,84 @@ <h3>Multi-Post Stories Gain+Glory</h3>
...
+      <section class="contact-section">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
 
+        <form action="https://formspree.io/f/xvolveaz" method="post">
+          
+          <ul>
+            <li><input type="text" name="first_name" placeholder="Full name" required maxlength="30"/></li>
+            <li><input type="email" name="email" placeholder="Email address" required/></li>
+            <li>
+            <textarea required maxlength="500">Enter text here</textarea>
```
Comment (awais-amjed) [inline index.html]: Kindly make the `Enter text here` `placeholder text` for a better user experience 🤗

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @mansa-susa , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🤗🥳🙌

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (mansa-susa) [reply index.html]: This is what the figma design specify

File: style.css (hunk 2 above)
Comment (mansa-susa) [reply style.css]: done.

File: index.html (hunk 1 above)
Comment (mansa-susa) [reply index.html:192]: implemented.

File: index.html (hunk 3 above)
Comment (Bondok6) [reply index.html]: - [ ] As the previous reviewer mentioned and according to the requirements all fields should have placeholder text, kindly put "Enter text here" inside the placeholder attribute 👌 
- [ ] And to match the Figma design you can style it by adding in your CSS file 💡
```
textarea::placeholder {
  color: #091e42;
  padding: 12px;
}
```

image

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @mansa-susa 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✅
- Great job using FormSpree ✅
- Using Gitflow correctly ✅
- Your project is professional ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @mansa-susa 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No Linter errors ✅

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**